tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -174,6 +174,13 @@ def validate_output(self, result, cert_data):
         }
         assert result.return_code == 0
         assert cert_data['success_message'] in result.stdout
+
+        # validate correct installer used
+        pattern, count = 'satellite-installer --scenario satellite', 2
+        assert len(re.findall(pattern, result.stdout)) == count
+        pattern, count = 'foreman-installer --scenario katello', 0
```

Comment:
```suggestion
        pattern, count = 'satellite-installer --scenario katello', 0
```

Reply:
This check is to make sure foreman-installer isn't being recommended.

Reply:
Oh sry, I missed BZ despcription.